tests/salt-tcpdump.py

```diff
@@ -66,7 +66,7 @@ def add_args(self):
             default="eth0",
             dest="iface",
             required=False,
-            help="the interface to dump themaster runs on(default:eth0)",
+            help="the interface to dump the master runs on(default:eth0)",
```

Comment:
**question** should `The` be capitalized?

Reply:
(the first The, i.e. `the interface` -> `The interface`

Reply:
Fixed on next push

Reply:
Do you mean you have a follow up PR that will fix this capitalization?

Reply:
Already fixed and pushed